erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -327,18 +390,21 @@ int ExternalOutput::deliverAudioData_(char* buf, int len) {
 int ExternalOutput::deliverVideoData_(char* buf, int len) {
   if (videoSourceSsrc_ == 0) {
     RtpHeader* h = reinterpret_cast<RtpHeader*>(buf);
-    videoSourceSsrc_ = h->getSSRC();
+    if (h->getPayloadType() == RED_90000_PT || h->getPayloadType() == VP8_90000_PT) {
```

Comment:
@jnoring agree, we should make it configurable in anycase. But I saw at some place that those issues remain even with FEC disabled?